Branch-SDK/Branch-SDK/BNCPreferenceHelper.m

```diff
@@ -34,6 +34,7 @@
 NSString * const BRANCH_PREFS_KEY_IS_REFERRABLE = @"bnc_is_referrable";
 NSString * const BRANCH_PREFS_KEY_BRANCH_UNIVERSAL_LINK_DOMAINS = @"branch_universal_link_domains";
 NSString * const BRANCH_REQUEST_KEY_EXTERNAL_INTENT_URI = @"external_intent_uri";
+NSString * const BRANCH_REQUEST_KEY_MIXPANEL_DISTINCT_ID = @"$mixpanel_distinct_id";
```

Comment:
should we include `$` in the key? I thought these are reserved for branch related attributes